content/docs/command-reference/data/status.md

```diff
@@ -1,6 +1,6 @@
 # data status
 
-Show changes in the data tracked by DVC in the workspace.
+Show changes to the files and directories tracked by DVC.
```

Comment:
The command name is `data`. I am not sure what we get by replacing it.

Reply:
We are specific about what we mean by data. Docs should explain 🙂